ipi/engine/smotion/remd.py

```diff
@@ -169,3 +184,6 @@ def step(self, step=None):
                     sf.write(" % 5d" % (i))
                 sf.write("\n")
 
+        info("# REMD step evaluated in %f (%f eval, %f swap) sec." % (time.time()-t_start, t_eval, t_swap), verbosity.debug)
```

Comment:
Ideally, we would have a more systematic timing mechanism throughout the code, I even have a timer class ready, but so far other things were more pressing. One of these days...

Reply:
Agreed.